tasks/push-rpm-data-to-pyxis/push-rpm-data-to-pyxis.yaml

```diff
@@ -176,21 +175,21 @@ spec:
           $UPLOAD_SCRIPT --retry --image-id "$IMAGEID" --sbom-path "$FILE" --verbose > "${IMAGEID}.out" 2>&1 &
 
           jobs+=($!)  # Save the background process ID
-          images+=($IMAGEID)
+          images+=("$IMAGEID")
           ((++count))
 
-          if [ $((count%N)) -eq 0 -o $((count)) -eq $total ]; then
```

Comment:
Why didn't it like this? I'm curious. If you remember. I thought this was perfectly ok.

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2166